src/components/subsetDraft/Classification.jsx

```diff
@@ -272,20 +281,19 @@ export const ClassificationInfo = ({id, info}) => {
         <div style={{backgroundColor: '#eff4f5'}}
              className='panel'>
             <Title size={4}>{t('Code list info')}</Title>
-            <Paragraph><strong>Id:</strong> {id}</Paragraph>
-            <table style={{border: 'none'}}>
-                <thead>
-                <th>{t('From')}</th>
-                <th>{t('To')}</th>
-                <th>{t('Version')}</th>
-                </thead>
+            <Paragraph><strong>Id:</strong> {id}</Paragraph>k
```

Comment:
typo? 'k' at the end of the line

Reply:
Ups... I do not have a mouse. I use my keyboard. Looks like I slipped here.... :D